front/src/modules/SimulationReportSheet/styles/SimulationReportStyleSheet.ts

```diff
@@ -394,6 +394,65 @@ const styles = {
       marginTop: '8',
     },
   }),
+  schedulesToDuplicate: StyleSheet.create({
+    schedulesToDuplicate: {
+      marginBottom: 15,
+      paddingLeft: 16,
+      borderTop: '1 solid #D3D1CF',
+    },
+    title: {
+      marginTop: '13',
```

Comment:
```suggestion
      marginTop: '12',
```
The value on the mockup is 12.